.github/workflows/test.yml

```diff
@@ -101,6 +101,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
 
+      # Temporarily setup QEMU so we can run aarch64 build tests
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: 'arm64,arm'
```

Comment:
Running all the tests under qemu is going to take a _long_ time.

Reply:
For reference: in z2jh we do the following, where `--platform` is passed during `docker build` is run by chartpress.

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/d16d44aa44d43a3ac34a8fdec4a94328e47e2546/.github/workflows/test-docker-build.yaml#L47-L57

---

@yuvipanda I think you should only specify arm64 and not arm64,arm - at least unless you wish to go for arm32 support as well. Typically, the support for arm64 is quite more common, and in z2jh for example we provide arm64 + amd64 but not arm.

Reply:
@consideRatio yay ty. Just switched it to arm64.

@minrk I'm trying to get it to at least run on arm64 now, and can't get it to run at all :( I think it's still just running on amd64 despite me passing platform, not sure why.